core/whistle-1.0.0/src/wh_cache.erl

```diff
@@ -143,6 +143,19 @@ start_link(Name, ExpirePeriod, Props) ->
                                    )
     end.
 
+-spec build_bindings(wh_proplist()) -> wh_proplist().
+build_bindings([[]]) -> document_bindings([[]]);
+build_bindings(BindingProps) ->
+    [database_binding() | document_bindings(BindingProps)].
```

Comment:
this here `[database_binding() | BindingProps].`
or remove the function `database_binding` and replace it with a macro.